doc/sphinx/source/intro.rst

```diff
@@ -20,6 +20,3 @@ algebraic interface for matrix-free operator representation and supports run-tim
 selection of implementations tuned for a variety of computational device types,
 including CPUs and GPUs. libCEED’s purely algebraic interface can unobtrusively be
 integrated in new and legacy software to provide performance portable interfaces.
-
```

Comment:
I thought it was nice to see the reference at the end of the page where it was used, instead of having the reader click on it to see it, but yes, this way it's cleaner.